src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -95,8 +95,7 @@ internal sealed class TupleTypeSymbol : WrappedNamedTypeSymbol
             Debug.Assert(elementNames.IsDefault || elementTypes.Length == elementNames.Length);
 
             int numElements = elementTypes.Length;
-
-            if (numElements <= 1)
+            if (numElements == 0)
```

Comment:
I am looking at this at the moment.

---
In reply to: [124151349](https://github.com/dotnet/roslyn/pull/20407#discussion_r124151349) [](ancestors = 124151349)